### PR TITLE
use v0.30.3 flow-go

### PR DESCRIPTION
**Pull request by zhangchiqing**
The bulk of PR changes are from the two actions explained in the comments.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,60 +13,61 @@ require (
...
+	golang.org/x/sync v0.1.0
+	google.golang.org/api v0.114.0
+	google.golang.org/grpc v1.53.0
 )
 
 require (
-	github.com/onflow/flow-go v0.29.17-read-leaf-nodes
-	github.com/onflow/flow-go/crypto v0.24.4
-	github.com/optakt/flow-dps-rosetta v1.0.0
+	github.com/onflow/flow-go v0.30.3-fix-read-leaf-nodes
```
Comment (zhangchiqing) [inline go.mod:30]: use the latest API from https://github.com/onflow/flow-go/pull/4213

File: service/tracker/execution_test.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"github.com/onflow/flow-go/engine/execution/computation/computer/uploader"
+	"github.com/onflow/flow-go/engine/execution/ingestion/uploader"
```
Comment (zhangchiqing) [inline service/tracker/execution_test.go:24]: this module was moved since v0.30

File: crypto_build.sh [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # crypto package
 PKG_NAME="github.com/onflow/flow-go/crypto"
 
-VERSION="v0.24.4"
+VERSION="v0.24.7"
```
Comment (zhangchiqing) [inline crypto_build.sh:6]: update to the latest version, otherwise it would cause inconsistent flow-go version

**Review by koko1123 — APPROVED** [review]
okay with merging, the auto annotation issues in this PR are going to be fixed by merging this:  #85

**Review by pattyshack — APPROVED** [review]

**Pull request merged**